src/manager.cpp

```diff
@@ -398,8 +398,7 @@ void Manager::collectSingleFruVpd(
 void Manager::deleteSingleFruVpd(
     const sdbusplus::message::object_path& i_dbusObjPath)
 {
-    // Dummy code to supress unused variable warning. To be removed.
-    logging::logMessage(std::string(i_dbusObjPath));
+    m_worker->deleteFruVpd(std::string(i_dbusObjPath));
```

Comment:
The parameter "i_dbusObjPath" is received by this API so the validity of that particular parameter needs to be checked by this API and then go ahead.

Reply:
Done